static/app/views/explore/tables/aggregateColumnEditorModal.tsx

```diff
@@ -129,7 +169,21 @@ function ColumnEditorRow({canDelete, column, onColumnDelete}: ColumnEditorRowPro
         icon={<IconGrabbable size="sm" />}
         {...listeners}
       />
-      {JSON.stringify(column.column)}
+      {isGroupBy(column.column) ? (
+        <GroupBySelector
+          groupBy={column.column}
+          groupBys={groupBys}
+          onChange={onColumnChange}
+          stringTags={stringTags}
+        />
+      ) : (
+        <VisualizeSelector
+          visualize={column.column}
+          onChange={onColumnChange}
+          numberTags={numberTags}
```

Comment:
I notice we pass these down a lot. Does the spans hook pull from a single context (i.e. it's cached after the first request)? I think there's a chance we can simplify some of the interfaces by just using the hook to get these tags